unittests/parser_test_case.py

```diff
@@ -84,3 +85,10 @@ def _test_calldef_exceptions(self, calldef, exceptions):
             (calldef.name,
              pprint.pformat([delc.name for delc in exception_decls]),
              pprint.pformat([delc.name for delc in exceptions_indeed])))
+
+if sys.version_info < (2, 7, 0):
```

Comment:
Ah ok, I don't have unittest2 installed, and was seeing `AttributeError`s in unit tests that used `assertIn()`. Added an additional check to do the above only if unittest2 is not imported.